src/devices/Lps25h/README.md

```diff
@@ -6,8 +6,40 @@ Some of the applications mentioned by the datasheet:
 - Weather station equipment
 - Sport watches
 
-See [samples](samples/README.md) for information about usage.
-
-## References
+## Documentation
 
 - https://www.st.com/resource/en/datasheet/lps25h.pdf
+
```

Comment:
while you are at adjusting those elements, make it with the proper pattern so something like:

```markdown
Official LPS25H documentation can be found [here](https://www.st.com/resource/en/datasheet/lps25h.pdf)
```

I've tried to adjust all those URLs as well